KarhooUISDK/UIComponents/LegalNotice/LegalNoticeStringBuilder.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  LegalNoticeStringBuilder.swift
+//  KarhooUISDK
+//
+//  Created by Bartlomiej Sopala on 31/01/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+public struct LegalNoticeStringBuilder {
```

Comment:
1. Why do we need a publicly accessible structure that has no public method? 
2. Please consider following namespace usage guide `https://google.github.io/swift/#nesting-and-namespacing`